### PR TITLE
give id to inputs

### PR DESCRIPTION
**Pull request by ZzzzHui**

### CODE REVIEW

File: internal/convenience/decoder/decoder.go [hunk 1]
```diff
@@ -110,6 +111,7 @@ func (o *OutputDecoder) HandleInput(
 		return fmt.Errorf("error getting converted input: %w", err)
 	}
 	_, err = o.convenienceService.CreateInput(ctx, &model.AdvanceInput{
+		ID:                     strconv.Itoa(input.Node.Index),
```
Comment (ZzzzHui) [inline internal/convenience/decoder/decoder.go:114]: I'm not too sure about this fix. What does this decoder do? Thanks

**Comment by ZzzzHui** [comment]
CI failed with error `level=error msg="thresholds on metrics 'checks' have been crossed"`
Not sure what's the error

File: api/reader.graphql [hunk 2]
```diff
@@ -87,13 +89,7 @@ type Voucher {
 "Top level queries"
 type Query {
   "Get input based on its identifier"
-  input(index: Int!): Input!
-  "Get a voucher based on its index"
-  voucher(voucherIndex: Int!, inputIndex: Int!): Voucher!
```
Comment (sandhilt) [inline api/reader.graphql:92]: > CI failed with error `level=error msg="thresholds on metrics 'checks' have been crossed"` Not sure what's the error

@ZzzzHui k6 and loadtest try queries on this "endpoints". Maybe we need revert schema this guys.

Comment (ZzzzHui) [reply api/reader.graphql:92]: is that an automatic test? is it possible to remove the tests for those queries that we don't need?

Comment (sandhilt) [reply api/reader.graphql:92]: My recommendation is to keep other queries and increment other.
Because removing them can be a mess.

Comment (ZzzzHui) [reply api/reader.graphql:92]: could you elaborate a bit more please.
What's "increment other"
and why removing them is a mess. It's a mess only for the CI or for other things?

**Review by fabiooshiro — APPROVED** [review]

File: api/reader.graphql (hunk 2 above)
Comment (ZzzzHui) [reply api/reader.graphql:92]: @sandhilt ⬆️

Comment (sandhilt) [reply api/reader.graphql:92]: Check this PR: #225

**Comment by ZzzzHui** [comment]
As discussed in today's meeting, the tests will adapt to the removal of top level queries for outputs. Thanks @sandhilt

**Pull request merged**